src/Language/ScriptToLanguage.php

```diff
@@ -7,15 +7,15 @@
 class ScriptToLanguage implements \Mpdf\Language\ScriptToLanguageInterface
 {
 
-	private $scriptDelimiterMap = [
+	private static $scriptDelimiterMap = [
 		'viet' => "\x{01A0}\x{01A1}\x{01AF}\x{01B0}\x{1EA0}-\x{1EF1}",
 		'persian' => "\x{067E}\x{0686}\x{0698}\x{06AF}",
 		'urdu' => "\x{0679}\x{0688}\x{0691}\x{06BA}\x{06BE}\x{06C1}\x{06D2}",
 		'pashto' => "\x{067C}\x{0681}\x{0685}\x{0689}\x{0693}\x{0696}\x{069A}\x{06BC}\x{06D0}", // ? and U+06AB, U+06CD
 		'sindhi' => "\x{067A}\x{067B}\x{067D}\x{067F}\x{0680}\x{0684}\x{068D}\x{068A}\x{068F}\x{068C}\x{0687}\x{0683}\x{0699}\x{06AA}\x{06A6}\x{06BB}\x{06B1}\x{06B3}",
 	];
 
-	private $scriptToLanguageMap = [
+	private static $scriptToLanguageMap = [
```

Comment:
Why static? Any particular reason for this? I think static should be avoided if possible, it is private so there are no benefits for this in class' non-static methods.

Reply:
It was a _very_ tiny memory optimisation while I was there. 
In fact, I wanted to turn this into a const at first, but private const are not available before PHP 7.1.
I'll rollback this change